src/follow-me/follow-me.js

```diff
@@ -0,0 +1,153 @@
+			<img className="activitypub-profile__avatar" src={ avatar } />
+			<div className="activitypub-profile__content">
+				<div className="activitypub-profile__name">{ name }</div>
+				<div className="activitypub-profile__handle">{ resource }</div>
+			</div>
+			<Follow profile={ profile } popupStyles={ popupStyles } userId={ userId } />
+		</div>
+	);
+}
+
+function Follow( { profile, popupStyles, userId } ) {
+	const [ isOpen, setIsOpen ] = useState( false );
+	const title = sprintf( __( 'Follow %s', 'activitypub' ), profile?.name );
+
+	return (
+		<>
+			<Button className="activitypub-profile__follow" onClick={ () => setIsOpen( true ) } >
+				{ __( 'Follow', 'activitypub' ) }
+			</Button>
+			{ isOpen && (
+				<Modal
+				className="activitypub-profile__confirm"
+				onRequestClose={ () => setIsOpen( false ) }
+				title={ title }
+				>
+					<Dialog profile={ profile } userId={ userId } />
+					<style>{ popupStyles }</style>
+			</Modal>
+			) }
+		</>
+	);
+}
+
+function Dialog( { profile, userId } ) {
+	const { resource } = profile;
+	const followText = __( 'Follow', 'activitypub' );
+	const loadingText = __( 'Loading...', 'activitypub' );
+	const openingText = __( 'Opening...', 'activitypub' );
+	const errorText = __( 'Error', 'activitypub' );
+	const [ buttonText, setButtonText ] = useState( followText );
+	const [ buttonIcon, setButtonIcon ] = useState( copy );
+	const ref = useCopyToClipboard( resource, () => {
+		setButtonIcon( check );
+		setTimeout( () => setButtonIcon( copy ), 1000 );
+	} );
+	const [ remoteProfile, setRemoteProfile ] = useState( '' );
+	const retrieveAndFollow = useCallback( () => {
+		const path = `/${ namespace }/users/${userId}/remote-follow?resource=${ remoteProfile }`;
+		setButtonText( loadingText );
+		apiFetch( { path } ).then( ( { url } ) => {
+			setButtonText( openingText );
+			setTimeout( () => {
+				window.open( url, '_blank' );
+				setButtonText( followText );
+			}, 200 );
+		} ).catch( () => {
+			setButtonText( errorText );
+			setTimeout( () => setButtonText( followText ), 2000 );
+		} );
+	}, [ remoteProfile ] );
+
+	return (
+		<div className="activitypub-follow-me__dialog">
+			<div className="apmfd__section">
+				<h4>{ __( 'Remote Follow', 'activitypub' ) }</h4>
+				<div className="apfmd-description">
+					{ __( 'Copy and paste this URL into the search field of your favourite Fediverse app or server.', 'activitypub' ) }
+				</div>
+				<div className="apfmd__button-group">
+					<input type="text" value={ resource } readOnly />
+					<Button ref={ ref }>
+						<Icon icon={ buttonIcon } />
+						{ __( 'Copy', 'activitypub' ) }
+					</Button>
+				</div>
+			</div>
+			<div className="apmfd__section">
+				<h4>{ __( 'Different Server', 'activitypub' ) }</h4>
+				<div className="apfmd-description">
+					{ __( 'Give us your Username/URL and we will start the process for you. (E.g. https://example.com/username or username@example.com)', 'activitypub' ) }
```

Comment:
Use URL/Username instead of Username/URL, to match the order of the examples (E.g ...)

```suggestion
					{ __( 'Give us your URL or Username and we will start the process for you. (E.g. https://example.com/username or username@example.com)', 'activitypub' ) }
```

Reply:
I actually decided not to name them at all in the end, but otherwise this was a good catch :)